src/controller/imageController.js

```diff
@@ -0,0 +1,62 @@
+var images=[];    // array of all inages
```

Comment:
please dont write any variables outside the angular scope. make factory and inject it to controllers/factories wherever you need.